subprojects/instant-execution/src/main/kotlin/org/gradle/instantexecution/DefaultInstantExecution.kt

```diff
@@ -141,6 +143,8 @@ class DefaultInstantExecution internal constructor(
             return
         }
 
+        buildScan.instantExecutionStoreAction()
```

Comment:
❓ does it have to be inside the _cache fingerprint zone_? otherwise I'd move it after `stopCollectingCacheFingerprint()`